test/test_pulsecal.py

```diff
@@ -206,13 +206,50 @@ def update_data(amp, ct):
         #save simulated data
         np.save(self.filename, ideal_data)
         # Verify output matches what was previously seen by matlab
-        pi_cal = cal.PiCalibration(self.q.label, numPulses)
+        pi_cal = cal.PiCalibration(self.q.label, numPulses, quad='imag')
```

Comment:
why this choice as default? The kernel is typically optimized to put the signal in the real quadrature

Reply:
No real reason.  I'm wondering if it's worth the time in CI to test this for all four quadratures.  The only down side is the amount of time it takes.

Reply:
If we have to chose one it should be real but I wanted to test one of the new quadratures as well.  I'll just add all four for now.

Reply:
Oh I didn't realize that this was just the test :)